src/man/sssd.conf.5.xml

```diff
@@ -722,6 +722,40 @@
                             </para>
                         </listitem>
                     </varlistentry>
+                    <varlistentry condition="build_passkey">
+                        <term>passkey_verification (string)</term>
+                        <listitem>
+                            <para>
+                                With this parameter the passkey verification
+                                can be tuned with a comma separated list of
+                                options. Supported options are:
+                                <variablelist>
+                                <varlistentry>
+                                    <term>user_verification=true|false</term>
```

Comment:
Typically it's denoted as ` (boolean)` in SSSD man pages.

Reply:
Fixed.